### PR TITLE
Create ci.yml

### PR DESCRIPTION
**Pull request by BeastyBlacksmith**
only merge when https://github.com/BeastyBlacksmith/pyama/pull/1 is green

### CODE REVIEW

**Comment by dwoschee** [comment]
Could you please rebase bbs/ci onto devel? Then I can accept the PR.
Thank you!

**Comment by BeastyBlacksmith** [comment]
Okay, I did the rebase, but there are a fiew files left, which I think I can delete, but would be good, if you can take a look before I do so.
Essentially I am talking about all *.py and *.bak files in this PR

**Comment by dwoschee** [comment]
Yes, there are some superfluous files in the PR. I think they come from commits bb538e46721bd845b23f98d962c2d62165768f23 to 35f73fb5c135e21b97e9d92dbeccd4763f9d2f53. As the commit messages indicate, the changes introduced with these commits are already included in the devel branch.

Would it be possible to rebase only commit 289ad1eb5859075f7e956ca2a616b77fe7c10320 and its children on the devel branch? This would be the cleanest way to remove the *.py and *.bak files.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  
+  tests:
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    name: "Python ${{ matrix.python-version }} on ${{ matrix.os }}"
+    runs-on: "${{ matrix.os }}"
+    
+    strategy:
+      fail-fast: true
+      matrix:
+        python-version: ["3.8"]
```
Comment (dwoschee) [inline .github/workflows/ci.yml:27]: Here, we could also include python 3.9.

**Comment by BeastyBlacksmith** [comment]
I think this is ready to merge

**Comment by dwoschee** [comment]
Sorry, forgot to squash …

**Pull request merged**